pkg/models/dashboards.go

```diff
@@ -141,7 +142,7 @@ func (d UpdatePluginDashboardError) Error() string {
 	return "Dashboard belong to plugin"
 }
 
-var (
+const (
```

Comment:
Great, can you please also remove `DashTypeJson` and `DashTypeScript` though? They're not in use.